### ISSUE TITLE
Other Widgets visually behind are not receiving mouse events

### ISSUE THREAD

**Issue by Uche01** [caduandrade/multi_split_view#62]
I have a use-case where I need to stack a `MultiSplitView` on top of another widget such that I could still interact with the widget below. Before `v3`, this was working until I upgraded. The reason for upgrade was to access the new `onDividerDragUpdate`

Below is a reproducible example:
```
class StackExample extends StatelessWidget {
  const StackExample({super.key});

@override
  Widget build(BuildContext context) {

    double leftPaneWidth = 300;
    double screenWidth = MediaQuery.of(context).size.width;
    double screenHeight = MediaQuery.of(context).size.height;
    double mainContainerWidth = screenWidth - leftPaneWidth;

    return Stack(
      children: [
        Positioned.fromRect(
          rect: Rect.fromLTWH(0, 0, leftPaneWidth, screenHeight),
          child: Center(
            child: ElevatedButton(
              onPressed: () { print("Button clicked"); },
              child: const Text('Click me!'),
            ), // I cannot interact with this button even when the widget stacked on it is wrapped with IgnorePointer
          ) 
        ),
        MultiSplitViewTheme(
          data: MultiSplitViewThemeData(
            dividerThickness: 4,
            dividerPainter: DividerPainters.background(color: Colors.blue)),
          child: MultiSplitView(
            initialAreas: [
              Area(size: leftPaneWidth, widget: IgnorePointer(child: Container())), // IgnorePointer widget. the Area object appears to override the effect of IgnorePointer
              Area(size: mainContainerWidth, widget: Container())
            ],
          ),
        ),
      ],
    );
  }
}
```

With the sample above, I cannot interact with the `button`, the pointer seems to be focused on the `Container` widget stacked on top of the button. I have also wrapped the `Container` widget with `IgnorePointer` but it didn't still work. It appears that the `Area` object wrapping the container widget is not letting the ignore pointer take effect.

PS: Swapping the `Stack` children positions appears to fix the issue, however this is a simple example. My usecase is much more complex, in my case, a swap fixed the interaction issue but resulted to another issue where I was unable to interact/drag the divider

**Comment by caduandrade** [comment]
You're wanting to use MultiSplitView as if it were transparent, right? Being able to click on everything behind it, right?

**Comment by Uche01** [comment]
> You're wanting to use MultiSplitView as if it were transparent, right? Being able to click on everything behind it, right?

Yes!

**Comment by caduandrade** [comment]
Version 3.0.2 released

**Comment by cassioso** [comment]
Wait, what if I don't want to let the user to click on everything behind a specific Area?

**Comment by caduandrade** [comment]
Do you want to block clicks? I think you could use an `AbsorbPointer` or `IgnorePointer`.

**Comment by helgoboss** [comment]
This change is somehow badly interacting with my application. It leads to negative side effects such as the macOS system alert sound to be played when I click certain widgets within the child or keyboard shortcuts not working.

I pinned it down to this particular line: `opaque: _draggingDivider != null` (see [line in commit](https://github.com/caduandrade/multi_split_view/commit/76f2f90351581d9423e9375a25866152eef7cb8c#diff-cfd68a369d512bcdc077b2ab088e0f6bf07455ea845bcec418634aefabeacc34R226)).

If I set `opaque` to `true` (which is also the default), things work again.

Maybe this could be made configurable?
